src/main/kotlin/specific/lotto/domain/WinningSet.kt

```diff
@@ -0,0 +1,12 @@
+package specific.lotto.domain
+
+data class WinningSet(val numbers: Set<Number>) {
```

Comment:
다음 글 중 `변수 이름에 자료형이 들어간다면?` 파트를 참고해보시면 어떨까요?

- https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming/

Reply:
부생성자를 활용하면 더 명시적으로 WinningSet 구현을 캡슐화할 수 있을 것 같습니다.

```suggestion
data class WinningSet(val numbers: Set<Number>) {

    constructor(numbers: List<Int>) : this(numbers.map { Number(it) }.toSet())
```

Reply:
부생성자를 사용하니 다른 곳에서 사용할 때 중복 코드가 훨씬 줄어들었습니다. 👍 
변수 명은 WinningNumbers로 수정했습니다.